gtk/PrefsDialog.cc

```diff
@@ -897,7 +898,8 @@ void NetworkPage::onCorePrefsChanged(tr_quark const key)
 {
     if (key == TR_KEY_peer_port)
     {
-        gtr_label_set_text(*portLabel_, _("Status unknown"));
+        gtr_label_set_text(*portIPv4Label_, _("Status unknown"));
+        gtr_label_set_text(*portIPv6Label_, _("Status unknown"));
```

Comment:
Two labels with initial "Status unknown" text feel like bad UI — it's not clear why there're two of them until you test the port and figure out that one is for IPv4 and another is for IPv6; didn't think of how but maybe it could still be a single label. There was also a report recently of v4 client connecting to v3 server, not sure how this will work and if we care.

Reply:
How about if I add `IPv4` and `IPv6` to the initial text as well? Not quite sure what is a good way to use 1 label to represent 2 results.

Reply:
I don't pretend to offer this as the best option, but one way would be e.g. this set of texts:
* Status: <b>unknown</b>
* Status: <b>checking…</b> (IPv4), <b>checking…</b> (IPv6)
* Status: <b>open</b> (IPv4), <b>checking…</b> (IPv6)
* Status: <b>checking…</b> (IPv4), <b>closed</b> (IPv6)
* Status: <b>closed</b> (IPv4), <b>open</b> (IPv6)

With e.g. these phrases for translation:
* Status: <b>{port_check_status}</b>
* Status: <b>{ipv4_port_check_status}</b> (IPv4), <b>{ipv6_port_check_status}</b> (IPv6)
* unknown
* checking…
* open
* closed
* error _(?)_

Reply:
And if the text feels too long, I'd be okay with moving the "Test port" button up so that it's on the same line as spin edit, which will potentially also improve accessibility if the label is made focusable, otherwise tab order might be weird (because I'd prefer the status label to come after the button in tab order, and going from right to left, in case of LTR languages, while forward-tabbing is highly unusual).

Reply:
fd8dd0a GTK
a5aa955 Qt

Added the "error" status.

a49b806 Also moved"Test port" button up as you suggested.